### ISSUE TITLE
Slow flight loop

### ISSUE THREAD

**Issue by MatzElectronics** [parallaxinc/Flight-Controller#36]
The current flight loop is 4ms - should it be increased to 5ms to accommodate the additional math required to GPS, and a few additional features?

**Comment by JasonDorie** [comment]
Done.  Not submitted, not flight tested, but IMU appears to be working
properly.

I've also reduced a bunch of the data being sent to GroundStation,
including the radio channels, the yaw / pitch / roll power values, counter,
and one or two others.  Since the update rate is now 20% slower, it means
we have 20% more time to send serial data.  With the reduction in packet
size, we should have more overhead for serial retries, and generally lower
packet loss.  It also doesn't hit the FTDI buffer thing as much either.

I figure since this version won't be back-compatible anyway, I might as
well break _everything_ at once.  I bumped this to version 3.0.0.

J
